zpl2.py

```diff
@@ -35,6 +35,32 @@
 ARG_COLOR = 'color'
 ARG_ROUNDING = 'rounding'
 ARG_DIAMETER = 'diameter'
+ARG_QRMODEL = 'qrmodel'
```

Comment:
IMO it would be best to follow [PEP-8](https://www.python.org/dev/peps/pep-0008/#constants) and use camel case for all these constants:

```
ARG_QR_MODEL
```

Reply:
Please call this constants `ARG_MODEL`, `ARG_MAGNIFICATION_FACTOR`, `ARG_CORRECTION_ERRROR` and `ARG_MASK_VALUE` to keep the naming of the [official ZPL2 documentation](https://www.zebra.com/content/dam/zebra/manuals/en-us/software/zpl-zbi2-pm-en.pdf).

Furthermore, some arguments, like magnification, are not specific to QR Code (Aztec and GS1 use magnification factor too).